setup.cfg

```diff
@@ -1,4 +1,8 @@
 [tool:pytest]
-testpaths = tests
+testpaths = tests/
+addopts = --cov=aiohttp_security --cov=tests
+asyncio_mode = auto
+xfail_strict = true
 filterwarnings=
     error
+    ignore::DeprecationWarning
```

Comment:
Ignoring all deprecation warnings wholesale is a bad idea because you'll miss when the new ones appear. They should be ignored temporarily on a case-by-case basis.

Reply:
Will be removed once updated to 3.11, too many warnings to deal with currently.

Reply:
I wish there was a way to make this Python-dependent. Here's an example of solving this problem via a local pytest plugin that extends the ignores dynamically: https://github.com/cherrypy/cheroot/blob/b137c4f/cheroot/test/_pytest_plugin.py.